src/WebRequest.cpp

```diff
@@ -792,14 +796,6 @@ void AsyncWebServerRequest::send(AsyncWebServerResponse* response) {
   if (_response)
     delete _response;
   _response = response;
-  if (_response == NULL) {
-    _client->close(true);
-    _onDisconnect();
-    _sent = true;
-    return;
-  }
```

Comment:
I've removed this weird use case where a user could just decide to close the connection by doing `send(null)`. IMO this is wrong because once a correctly formed http request goes through, it should be handled as a http request and a response should be sent.

closing the connection eagerly would also break the middleware chain since middleware acting after the handler is executing won't have any effect.

Reply:
`send(null)` now has the same effect has an empty handler:

```c++
  // curl -v -X GET http://192.168.4.1/handler-not-sending-response
  server.on("/handler-not-sending-response", HTTP_GET, [](AsyncWebServerRequest* request) {
    // handler forgot to send a response to the client => 501 Not Implemented
  });
```

Reply:
Looks good to me!